lib/movable_ink/aws/route53.rb

```diff
@@ -13,18 +13,20 @@ def resource_record_sets(hosted_zone_id, client = nil)
       end
 
       def get_resource_record_sets_by_instance_name(zone, instance_name, client = nil)
-        resource_record_sets(zone, client).select{|rrs| rrs.set_identifier == instance_name}.first.to_h
+        resource_record_sets(zone, client).select{|rrs| rrs.set_identifier == instance_name}.map(&:to_h)
       end
 
       def delete_resource_record_sets(zone, instance_name, client = nil)
-        resource_record_set = get_resource_record_sets_by_instance_name(zone, instance_name, client)
-        return if resource_record_set.empty?
+        resource_record_sets = get_resource_record_sets_by_instance_name(zone, instance_name, client)
+        return if resource_record_sets.empty?
 
         change_batch = {
-          "changes": [{
-            "action": 'DELETE',
-            "resource_record_set": resource_record_set
-          }]
+          "changes": resource_record_sets.map { |resource_record_set|
```

Comment:
The limit for resource_record_sets in change_resource_record_sets request body is pretty high so I think we can omit checking the lengths change_batch.

https://docs.aws.amazon.com/Route53/latest/DeveloperGuide/DNSLimitations.html#limits-api-requests-changeresourcerecordsets

Reply:
👍

Reply:
Yeah, I can't imagine us registering dozens of dns records of the same machine or anything.